src/components/UniversalNavbarExpanded/MobileNav/MobileNav.module.scss

```diff
@@ -51,6 +51,21 @@ $navbar-border: solid 1px var(--GrayStrokeAndDisabled--translucent);
       }
     }
   }
+
+  &.singleCta {
+    :global {
+      @include for-phone-and-tablet {
+        .cta-button.show-when-scrolled.navbar-expanded {
+          right: 7px;
+        }
+      }
+      @media (max-width: $phone-range-small-end) {
+        .cta-button.show-when-scrolled.navbar-expanded {
+          right: 7px;
```

Comment:
is this not covered by `for-phone-and-tablet`?

Reply:
I believe I added this to makeup for the hamburger nav not being shown in this variation, but will review locally again.

Reply:
Ah yeah, not sure why I duplicated this here. Developer error, good catch.